_commands/os_x/open.md

```diff
@@ -1,27 +1,23 @@
-#Open (OSX)
+---
+---
 
-`open` is used to open files, directories or URLs from the terminal. 
-
-###Options
-
-Here are some useful (but not all) options for `open` (Adapted from the `man` page):
-
-|     Flag     |               Description                |
-| :----------: | :--------------------------------------: |
-|      -a      |  Opens item with a specific application  |
-|      -e      |         Opens item with TextEdit         |
-|      -t      |   Opens item with default text editor    |
-|      -f      | Reads input from standard input and opens it with TextEdit |
-| -R, --reveal | Selects in the Finder instead of opening |
-|  -n, --new   | Opens a new instance of the application (even if it's already running) |
-|  -j, --hide  |         Launches the app hidden          |
+OPEN (OSX)
```

Comment:
lowercase all text and remove '(OS)' completely, its already in the section name